libs/utils/website/src/lib/pages/subscribe-page.tsx

```diff
@@ -7,7 +7,7 @@ import {ComponentProps} from 'react'
 import {ssrAuthLink} from '../auth-link'
 import {getSessionTokenProps} from '../get-session-token-props'
 
-type SubscribePageProps = Omit<ComponentProps<typeof SubscribeContainer>, ''>
+export type SubscribePageProps = Omit<ComponentProps<typeof SubscribeContainer>, ''>
```

Comment:
```suggestion type SubscribePageProps = Omit<ComponentProps<typeof SubscribeContainer>, ''>
```

Reply:
oh damn, was menat to revert back to `type SubscribePageProps = Omit<ComponentProps<typeof SubscribeContainer>, ''>` not fully remove it.